src/plopp/graphics/line.py

```diff
@@ -125,9 +125,8 @@ def _preprocess_hist(self, data: dict) -> Tuple[dict, bool]:
             if data["mask"] is not None:
                 data["mask"]["y"] = np.concatenate(
                     (data["mask"]["y"][0:1], data["mask"]["y"]))
-            data["variances"]["x"] = 0.5 * (x[1:] + x[:-1])
-        else:
-            data["variances"]["x"] = x
+        if self._data.variances is not None:
+            data["variances"]["x"] = 0.5 * (x[1:] + x[:-1]) if hist else x
```

Comment:
Um, what is this doing?

Reply:
It's placing the error bar in the middle of the bin instead of on the edges, when the coord is bin edges and the line is a step function.

Reply:
Ok, so the "x" is the "x" position of the error bars, not the "x" coordinate?

Reply:
yes